src/redux/StudentEdit.redux/StudentEdit.action.js

```diff
@@ -0,0 +1,17 @@
+import Axios from 'axios';
+
+export const EDIT_USER_START = 'EDIT_USER_START';
+export const EDIT_USER_SUCCESS = 'EDIT_USER_SUCCESS';
+export const EDIT_USER_FAILURE = 'EDIT_USER_FAILURE';
+
+export const editUser = (id, { user_id, name }) => (dispatch) => {
```

Comment:
I'd actually skip the axios stuff for the moment, have it just edit the local mock data instead. the overall structure of this action looks good though. let me know if you want a hand doing the mock-edits.

Reply:
took out axios